src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java

```diff
@@ -241,23 +241,33 @@ private void checkRepresentative(Appeal appeal, Map<String, Object> ocrCaseData,
         }
     }
 
-    private void checkMrnDetails(Appeal appeal) {
+    private void checkMrnDetails(Appeal appeal, Map<String, Object> ocrCaseData) {
+
+        String dwpIssuingOffice = getDwpIssuingOffice(appeal, ocrCaseData);
+
         if (!doesMrnDateExist(appeal)) {
             warnings.add(getMessageByCallbackType(callbackType, "", MRN_DATE, IS_EMPTY));
         } else {
             checkDateValidDate(appeal.getMrnDetails().getMrnDate(), MRN_DATE, "", true);
         }
 
-        if (!doesIssuingOfficeExist(appeal)) {
-            warnings.add(getMessageByCallbackType(callbackType, "", ISSUING_OFFICE, IS_EMPTY));
-
-        } else if (appeal.getBenefitType() != null && appeal.getBenefitType().getCode() != null) {
+        if (dwpIssuingOffice != null && appeal.getBenefitType() != null && appeal.getBenefitType().getCode() != null) {
 
-            Optional<OfficeMapping> officeMapping = dwpAddressLookupService.getDwpMappingByOffice(appeal.getBenefitType().getCode(), appeal.getMrnDetails().getDwpIssuingOffice());
+            Optional<OfficeMapping> officeMapping = dwpAddressLookupService.getDwpMappingByOffice(appeal.getBenefitType().getCode(), dwpIssuingOffice);
 
             if (!officeMapping.isPresent()) {
                 warnings.add(getMessageByCallbackType(callbackType, "", ISSUING_OFFICE, IS_INVALID));
             }
+        } else if (dwpIssuingOffice == null) {
+            warnings.add(getMessageByCallbackType(callbackType, "", ISSUING_OFFICE, IS_EMPTY));
+        }
+    }
+
+    private String getDwpIssuingOffice(Appeal appeal, Map<String, Object> ocrCaseData) {
+        if (!doesIssuingOfficeExist(appeal)) {
```

Comment:
I think logic should be if !ocrCaseData.isEmpty() then getField(ocrCaseData, "office")